cmd/src/setup.rs

```diff
@@ -95,19 +86,38 @@ pub fn initial_logger(config: &TiKvConfig) {
                     e
                 );
             });
-            let drainer = logger::LogDispatcher::new(drainer, slow_log_drainer);
-            logger::init_log(
-                drainer,
-                config.log_level,
-                true,
-                true,
-                vec![],
-                config.slow_log_threshold.as_millis(),
+            Some(slow_log_drainer)
+        };
+        let rocksdb_log_drainer = if config.rocksdb.info_log_dir.is_empty() {
+            None
```

Comment:
if not set, should we write to same dir as `config.log_file`?  @yiwu-arbug

Reply:
yes

Reply:
hmm, I didn't realize tikv config specifies `log-file` but not `log-dir`. let's by default still put the logs in rocksdb dir.